cloud/src/test/scala/org/apache/spark/cloud/AzureInstantiationSuite.scala

```diff
@@ -0,0 +1,29 @@
+/*
```

Comment:
They don't detect much; they're just a simple sanity check of dependency pull in. Easy to pull; it just means that all tests will go downstream.

My downstream tests [are more rigorous](https://github.com/steveloughran/spark-cloud-examples/blob/master/cloud-examples/src/test/scala/com/hortonworks/spark/cloud/s3/S3DependencyCheckSuite.scala) . and can also be executed under Jenkins without credentials, so there's nothing lost in cutting these, except that PRs to the scala project won't show up the problems directly